packages/EasyPagination/src/Traits/DoctrinePaginatorTrait.php

```diff
@@ -43,7 +43,16 @@ public function getTotalItems(): int
         }
 
         $countAlias = \sprintf('_count%s', $this->fromAlias ? \sprintf('_%s', $this->fromAlias) : '');
-        $queryBuilder = $this->createQueryBuilder()->select(\sprintf('COUNT(1) as %s', $countAlias));
+
+        $queryBuilder = $this->createQueryBuilder()->select(
+            \sprintf(
+                'COUNT(DISTINCT %s) as %s', // Need to distinct to remove duplicates caused by joins.
+                $this->fromAlias,
```

Comment:
`fromAlias` is nullable, what happen when it is not set? Should we default it to something?

Reply:
okay, gonna test what should be the proper default if fromAlias is null.

Reply:
Keep me posted 😉

Reply:
Having hard time with this, null alias is only when this trait is used on DoctrineDbalLengthAwarePaginator. Checking EasyRepository package only have AbstractPaginatedDoctrineOrmRepository, that only DoctrineOrmLengthAwarePaginator.

Reply:
`eonx-com/easy-async` is using the dbal paginator, you can test it there.